workbench

```diff
@@ -181,6 +181,7 @@ def create():
         if "field_member_of" in row.keys() and (
             len(row["field_member_of"]) > 0
             and value_is_numeric(row["field_member_of"]) is False
+            and config["subdelimiter"] not in row["field_member_of"]
```

Comment:
@joecorall I've added full support for multiple `field_member_of` values [here](https://github.com/mjordan/islandora_workbench/blob/main/workbench#L184-L213) as lack of checking for the parent nodes broke our automated population of our IR. Your line 184 will need to be removed before we merge the CI test changes.

Reply:
@mjordan i removed it. That test will probably fail now. I guess I'll remove the test, too.